sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraIO.java

```diff
@@ -81,6 +83,7 @@
  *        .withEntity(Person.class));
  * }</pre>
```

Comment:
Please add a note to the javadoc itself explaining why it is marked experimental, and linking to a JIRA (create if necessary) about establishing continuously running IT coverage for CassandraIO.